test/common.py

```diff
@@ -647,10 +647,9 @@ def metafunc(self, mode, *args, **kwargs):
       self.emcc_args.append('-fwasm-exceptions')
       self.set_setting('SUPPORT_LONGJMP', 'wasm')
       if mode == 'wasm':
```

Comment:
Should we also change the mode names to something like 'legacy' and 'exnref'?

Reply:
Yeah test suffixes / mode names will be changed too in a followup.